web/package.json

```diff
@@ -25,6 +25,7 @@
     "@types/web": "^0.0.96",
     "@typescript-eslint/eslint-plugin": "^5.49.0",
     "@typescript-eslint/parser": "^5.49.0",
+    "arraybuffer-loader": "^1.0.8",
```

Comment:
this dependency is `save-dev` only. It's for loading the class files on the browser.